tests/sles4sap/sap_deployment_automation_framework/create_deployer_vm.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Test module for creating SDAF Deployer VM by cloning pre-deployed OS snapshot, containing all SDAF tools.
+
+# Required OpenQA variables:
+# 'SDAF_DEPLOYER_RESOURCE_GROUP' Existing deployer resource group - part of the permanent cloud infrastructure.
+
+# Optional:
+# 'SDAF_DEPLOYER_SNAPSHOT' define existing snapshot name to be used as a source
```

Comment:
maybe add a small note about the fact that snapshot is not created by this test itself but it is a prerequisite to run the test.

Reply:
I expanded the Summary section: 
```
# Summary: Test module for creating SDAF Deployer VM by cloning existing OS snapshot, containing all SDAF tools.
#   Snapshot is a permanent clone of original Deployer disk and must be created beforehand. 
#   This test module expects it to already exists with default name 'deployer_snapshot_latest' or defined by 
#   OpenQA parameter SDAF_DEPLOYER_SNAPSHOT
```